### PR TITLE
Tables generating + postmap

### PR DESCRIPTION
**Pull request by VeselaHouba**
Thought this might be useful. 

Also edited the readme a bit, as it felt a too complex for single header.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +30,26 @@ will produce
...
+### Postfix lookup tables
+
+You can also create multiline [tables](http://www.postfix.org/DATABASE_README.html#types) using `POSTMAP_<filename>` like this example:
+```
+environment:
+  - POSTFIX_transport_maps=hash:/etc/postfix/transport
+  - |
+    POSTMAP_transport=gmail.com smtp
+    mydomain.com relay:[relay1.mydomain.com]:587
+    * relay:[relay2.mydomain.com]:587
```
Comment (wader) [inline README.md:42]: Nice, didn't know about this syntax

**Comment by wader** [comment]
Thanks, like it! and i think it should be safe, will not affect any existing config?

**Comment by VeselaHouba** [comment]
> Thanks, like it! and i think it should be safe, will not affect any existing config if no POSTMAP_* is defined?

No, it has no effect if not defined. Unless someone purposely overwrites existing file like `POSTMAP_master.cf` then it should be safe and backward-compatible.

File: README.md (hunk 1 above)
Comment (VeselaHouba) [reply README.md:42]: I just found out today 😄  myself

**Comment by wader** [comment]
Thanks!

**Pull request merged**